mountpoint-s3/tests/fuse_tests/fork_test.rs

```diff
@@ -158,11 +158,7 @@ fn run_in_background_fail_on_mount() -> Result<(), Box<dyn std::error::Error>> {
     let mount_point = assert_fs::TempDir::new()?;
 
     let mut cmd = Command::cargo_bin("mount-s3")?;
-    cmd.arg(&bucket)
-        .arg(mount_point.path())
-        .arg("--auto-unmount")
-        .spawn()
```

Comment:
Seems like we were spawning same command twice, one here and one in L170. I've removed spawn from this line. Same case with the other changed test case in this file.